--- quota-tools/Makefile~	2013-02-11 01:51:03.064103079 +0100
+++ quota-tools/Makefile	2013-02-11 01:51:06.201145397 +0100
@@ -1,4 +1,5 @@
 PROGS         = quotacheck quotaon quota quot repquota warnquota quotastats xqmstats edquota setquota convertquota rpc.rquotad quotasync quota_nld
+#include <rpc/xdr.h>
 SOURCES       = bylabel.c common.c convertquota.c edquota.c pot.c quot.c quota.c quotacheck.c quotacheck_v1.c quotacheck_v2.c quotaio.c quotaio_rpc.c quotaio_v1.c quotaio_v2.c quotaio_tree.c quotaio_xfs.c quotaio_meta.c quotaio_generic.c quotaon.c quotaon_xfs.c quotaops.c quotastats.c quotasys.c repquota.c rquota_client.c rquota_server.c rquota_svc.c setquota.c warnquota.c xqmstats.c svc_socket.c quotasync.c
 CFLAGS        = -O2 -Wa,--compress-debug-sections -gdwarf-4 -fvar-tracking-assignments -frecord-gcc-switches -Wstrict-aliasing=2 -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector --param=ssp-buffer-size=4 -fstack-protector-all -fPIC -fPIE -fPIE -D_GNU_SOURCE -Wall -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 CPPFLAGS      = -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include  
@@ -113,34 +114,34 @@ quotaon: quotaon.o quotaon_xfs.o $(LIBOB
 quotacheck: quotacheck.o quotacheck_v1.o quotacheck_v2.o quotaops.o $(LIBOBJS)
 	$(CC) $(LDFLAGS) -o $@ $^ $(EXT2LIBS)
 
-quota: quota.o quotaops.o $(LIBOBJS)
+quota: quota.o quotaops.o $(LIBOBJS) -ltirpc
 
-quotasync: quotasync.o $(LIBOBJS)
+quotasync: quotasync.o $(LIBOBJS) -ltirpc
 
-quot: quot.o $(LIBOBJS)
+quot: quot.o $(LIBOBJS) -ltirpc
 
-repquota: repquota.o $(LIBOBJS)
+repquota: repquota.o $(LIBOBJS) -ltirpc
 
-warnquota: warnquota.o $(LIBOBJS)
+warnquota: warnquota.o $(LIBOBJS) -ltirpc
 	$(CC) $(LDFLAGS) -o $@ $^ $(LDAPLIBS)
 
 quotastats: quotastats.o common.o pot.o
 
 xqmstats: xqmstats.o common.o pot.o
 
-edquota: edquota.o quotaops.o $(LIBOBJS)
+edquota: edquota.o quotaops.o $(LIBOBJS) -ltirpc
 
-setquota: setquota.o quotaops.o $(LIBOBJS)
+setquota: setquota.o quotaops.o $(LIBOBJS) -ltirpc
 
-convertquota: convertquota.o $(LIBOBJS)
+convertquota: convertquota.o $(LIBOBJS) -ltirpc
 
-rpc.rquotad: rquota_server.o rquota_svc.o svc_socket.o $(LIBOBJS)
+rpc.rquotad: rquota_server.o rquota_svc.o svc_socket.o $(LIBOBJS) -ltirpc
 	$(CC) $(LDFLAGS) -pie -o $@ $^ $(LIBS)
 	@if readelf -d $@|fgrep -q TEXTREL; then echo "*** Text relocation"; false; else true; fi
 
 ifneq ($(NETLINKLIBS),)
 quota_nld: quota_nld.o $(LIBOBJS)
-	$(CC) $(LDFLAGS) -o $@ $^ $(NETLINKLIBS)
+	$(CC) $(LDFLAGS) -o $@ $^ $(NETLINKLIBS) -ltirpc
 endif
 
 pot.o: pot.c pot.h
--- quota-tools/Makefile.in~	2013-02-11 01:44:50.440052550 +0100
+++ quota-tools/Makefile.in	2013-02-11 01:45:47.003823776 +0100
@@ -108,7 +108,7 @@ endif
 	-$(INSTALL) -m $(DEF_MAN_MODE) *.3 $(ROOTDIR)$(mandir)/man3
 	-$(INSTALL) -m $(DEF_MAN_MODE) *.8 $(ROOTDIR)$(mandir)/man8
 
-quotaon: quotaon.o quotaon_xfs.o $(LIBOBJS)
+quotaon: quotaon.o quotaon_xfs.o $(LIBOBJS) -ltirpc
 
 quotacheck: quotacheck.o quotacheck_v1.o quotacheck_v2.o quotaops.o $(LIBOBJS)
 	$(CC) $(LDFLAGS) -o $@ $^ $(EXT2LIBS) -ltirpc
--- quota-tools/rquota.h~	2013-02-11 01:29:31.700475678 +0100
+++ quota-tools/rquota.h	2013-02-11 01:41:15.234086838 +0100
@@ -7,6 +7,7 @@
 #define _RQUOTA_H_RPCGEN
 
 #include <rpc/rpc.h>
+#include <rpc/xdr.h>
 
 
 #ifdef __cplusplus
