diff -up quota-4.06/warnquota.c.1~ quota-4.06/warnquota.c
--- quota-4.06/warnquota.c.1~	2019-11-05 16:12:59.000000000 +0100
+++ quota-4.06/warnquota.c	2022-06-19 02:10:45.422843834 +0200
@@ -883,7 +883,7 @@ cc_parse_err:
 			}
 #ifdef USE_LDAP_MAIL_LOOKUP
 			else if (!strcmp(var, "LDAP_MAIL")) {
-				if(strcasecmp(value, "true") == 0) 
+				if(strncasecmp(value, "true", 4) == 0) 
 					config->use_ldap_mail = 1;
 				else
 					config->use_ldap_mail = 0;
diff -up quota-4.06/warnquota.conf.1~ quota-4.06/warnquota.conf
--- quota-4.06/warnquota.conf.1~	2019-09-12 10:19:47.000000000 +0200
+++ quota-4.06/warnquota.conf	2022-06-19 02:11:23.601068338 +0200
@@ -19,7 +19,7 @@ FAIL        = "configure /etc/warnquota.
 #
 # command used for sending mails
 #
-MAIL_CMD = "/usr/lib/sendmail -t"
+MAIL_CMD = "/usr/bin/sendmail -t"
 
 #
 # Standard mail fields
@@ -40,11 +40,11 @@ PHONE 		= "123 456 789"
 # The expressions %i, %h, %d, and %% are substituted for user/group name,
 # host name, domain name, and '%' respectively. For backward compatibility
 # %s behaves as %i but is deprecated.
-MESSAGE         = Hello user %i, I've noticed you use too much space\
- on my disk in %h.%d.|Delete your files on the following filesystems:|
+MESSAGE         = Your disk usage has exceeded the agreed limits\
+ on this machine.|Please delete any unnecessary files on the following filesystems:|
 # Text in the end of the mail (if not specified, default text using SUPPORT and PHONE
 # is created)
-SIGNATURE	= See you!|			Your admin of %h|
+SIGNATURE	= root@example.com
 # Following text is used for mails about group exceeding quotas
 GROUP_MESSAGE	= Hello,|\
 your group %i is using too much disk space at %h.|\
