--- quota-tools/Makefile.in.tirpc~	2013-07-05 06:03:31.477344461 +0200
+++ quota-tools/Makefile.in	2013-07-05 06:06:41.113790010 +0200
@@ -8,6 +8,7 @@ RPCSRC        = rquota.h rquota_xdr.c rq
 LIBS          = @LIBS@
 LDFLAGS       = @LDFLAGS@
 LDAPLIBS      = @LDAPLIBS@
+RPCLIB        = -ltirpc
 
 INSTALL       = @INSTALL@
 STRIP         = @STRIP@
@@ -108,39 +109,39 @@ endif
 	-$(INSTALL) -m $(DEF_MAN_MODE) *.3 $(ROOTDIR)$(mandir)/man3
 	-$(INSTALL) -m $(DEF_MAN_MODE) *.8 $(ROOTDIR)$(mandir)/man8
 
-quotaon: quotaon.o quotaon_xfs.o $(LIBOBJS)
+quotaon: quotaon.o quotaon_xfs.o $(LIBOBJS) $(RPCLIB)
 
 quotacheck: quotacheck.o quotacheck_v1.o quotacheck_v2.o quotaops.o $(LIBOBJS)
-	$(CC) $(LDFLAGS) -o $@ $^ $(EXT2LIBS)
+	$(CC) $(LDFLAGS) -o $@ $^ $(EXT2LIBS) $(RPCLIB)
 
-quota: quota.o quotaops.o $(LIBOBJS)
+quota: quota.o quotaops.o $(LIBOBJS) $(RPCLIB)
 
-quotasync: quotasync.o $(LIBOBJS)
+quotasync: quotasync.o $(LIBOBJS) $(RPCLIB)
 
-quot: quot.o $(LIBOBJS)
+quot: quot.o $(LIBOBJS) $(RPCLIB)
 
-repquota: repquota.o $(LIBOBJS)
+repquota: repquota.o $(LIBOBJS) $(RPCLIB)
 
 warnquota: warnquota.o $(LIBOBJS)
-	$(CC) $(LDFLAGS) -o $@ $^ $(LDAPLIBS)
+	$(CC) $(LDFLAGS) -o $@ $^ $(LDAPLIBS) $(RPCLIB)
 
 quotastats: quotastats.o common.o pot.o
 
 xqmstats: xqmstats.o common.o pot.o
 
-edquota: edquota.o quotaops.o $(LIBOBJS)
+edquota: edquota.o quotaops.o $(LIBOBJS) $(RPCLIB)
 
-setquota: setquota.o quotaops.o $(LIBOBJS)
+setquota: setquota.o quotaops.o $(LIBOBJS) $(RPCLIB)
 
-convertquota: convertquota.o $(LIBOBJS)
+convertquota: convertquota.o $(LIBOBJS) $(RPCLIB)
 
-rpc.rquotad: rquota_server.o rquota_svc.o svc_socket.o $(LIBOBJS)
+rpc.rquotad: rquota_server.o rquota_svc.o svc_socket.o $(LIBOBJS) $(RPCLIB)
 	$(CC) $(LDFLAGS) -pie -o $@ $^ $(LIBS)
 	@if readelf -d $@|fgrep -q TEXTREL; then echo "*** Text relocation"; false; else true; fi
 
 ifneq ($(NETLINKLIBS),)
 quota_nld: quota_nld.o $(LIBOBJS)
-	$(CC) $(LDFLAGS) -o $@ $^ $(NETLINKLIBS)
+	$(CC) $(LDFLAGS) -o $@ $^ $(NETLINKLIBS) $(RPCLIB)
 endif
 
 pot.o: pot.c pot.h
